products/cpdb/sql/projectscategorization.sql

```diff
@@ -5,810 +5,705 @@
 --Group projects into the ITT, Vehicles, and Equipment category
 --managing agency: ALL
 --SELECT * FROM cpdb_dcpattributes
-UPDATE cpdb_dcpattributes SET typecategory = 'ITT, Vehicles, and Equipment'
+UPDATE cpdb_dcpattributes
+SET typecategory = 'ITT, Vehicles, and Equipment'
 WHERE (
-upper(description) LIKE '%VEHICLE%'
-OR upper(description) LIKE '%AMBULANCE%'
-OR upper(description) LIKE '%BUSES%'
-OR upper(description) LIKE '%TRUCK%'
-OR upper(description) LIKE '%TRAILER%'
-OR upper(description) LIKE '%VANS%'
+    upper(description) LIKE any(
+        ARRAY[
+            '%VEHICLE%',
```

Comment:
Maybe worth just putting these in a separate file to be read in? Or in the `data` folder and loaded as a table? That actually maybe makes the most sense.